### ISSUE TITLE
Failed Build: /usr/bin/ld: unrecognised emulation mode: llvm

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#349]
Build: https://github.com/plentico/plenti/actions/runs/11598535151/job/32294588004

Error:
```
/usr/bin/ld: unrecognised emulation mode: llvm
Supported emulations: elf_x86_64 elf32_x86_64 elf_i386 elf_iamcu i386pep i386pe
clang++: error: linker command failed with exit code 1 (use -v to see invocation)

Error: The process '/opt/hostedtoolcache/goreleaser-action/2.3.2/x64/goreleaser' failed with exit code 1
```

This is the same error documented here: https://github.com/plentico/osxcross-target/issues/6

The problem is it's trying to use the system's `ld` but should be using the osxcross-target `ld`. Simply adding it to the path should fix this:
```
export PATH=/home/runner/work/osxcross/target/bin:$PATH
```

**Comment by jimafisk** [comment]
Still failing: https://github.com/plentico/plenti/actions/runs/11598677370/job/32295055197

So tried https://github.com/plentico/plenti/commit/0255b9934fe77ba2dec6cc26be646a3ac1f24bfa but that wasn't right either: https://github.com/plentico/plenti/actions/runs/11598825245/job/32295558415

```
error=hook failed: shell: 'export PATH=/home/runner/work/osxcross/target/bin:$PATH': exec: "export": executable file not found in $PATH: [no output]
Error: The process '/opt/hostedtoolcache/goreleaser-action/2.3.2/x64/goreleaser' failed with exit code 1
```

**Comment by jimafisk** [comment]
Should be fixed in [v0.7.14](https://github.com/plentico/plenti/releases/tag/v0.7.14)
